src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/Json/DataContractJsonSerializer.cs

```diff
@@ -171,37 +172,37 @@ internal static DataContract GetDataContract(DataContract declaredTypeContract,
             return contract;
         }
 
-        public override void WriteObject(Stream stream, object graph)
+        public override void WriteObject(Stream stream, object? graph)
         {
             _serializer.WriteObject(stream, graph);
         }
 
-        public override void WriteObject(XmlWriter writer, object graph)
+        public override void WriteObject(XmlWriter writer, object? graph)
         {
             _serializer.WriteObject(writer, graph);
         }
 
-        public override void WriteObject(XmlDictionaryWriter writer, object graph)
+        public override void WriteObject(XmlDictionaryWriter writer, object? graph)
         {
             _serializer.WriteObject(writer, graph);
         }
 
-        public override object ReadObject(Stream stream)
+        public override object? ReadObject(Stream stream)
         {
             return _serializer.ReadObject(stream);
         }
 
-        public override object ReadObject(XmlReader reader)
+        public override object? ReadObject(XmlReader reader)
         {
             return _serializer.ReadObject(reader);
         }
 
-        public override object ReadObject(XmlReader reader, bool verifyObjectName)
+        public override object? ReadObject(XmlReader reader, bool verifyObjectName)
         {
             return _serializer.ReadObject(reader, verifyObjectName);
         }
 
-        public override object ReadObject(XmlDictionaryReader reader)
+        public override object? ReadObject(XmlDictionaryReader reader)
```

Comment:
not sure if it affects any public APIs but serializerKnownTypeList below should be nullable

Reply:
```C#
private List<Type> GetKnownTypesFromContext(XmlObjectSerializerContext context, IList<Type> serializerKnownTypeList)
```

That method is never called. I added the annotation.